src/components/Frame/components/Loading/tests/Loading.test.tsx

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
 import {mountWithAppProvider} from 'test-utilities/legacy';
+import {animationFrame} from '@shopify/jest-dom-mocks';
+import {act} from 'react-dom/test-utils';
 import {Loading} from '../Loading';
 
 describe('<Loading />', () => {
-  const loading = mountWithAppProvider(<Loading />);
+  beforeEach(() => {
+    animationFrame.mock();
+  });
+
+  afterEach(() => {
+    animationFrame.restore();
+  });
+
+  it('increases over time', () => {
+    const loading = mountWithAppProvider(<Loading />);
 
-  it('mounts', () => {
-    expect(loading.exists()).toBe(true);
+    for (let i = 0; i <= 100; i++) {
```

Comment:
is this a good approach?
we can also change `100` to `100,000` to have a 100% code coverage but it took ~7 seconds in my computer...